modules/push/src/main/PushApi.scala

```diff
@@ -71,6 +72,13 @@ private final class PushApi(
     }
   }
 
+  def newMessage(t: Thread, p: Post): Funit =
+    pushToAll(t.receiverOf(p), _.challenge.create, PushApi.Data(
+      title = s"New message from ${t.senderOf(p)}",
```

Comment:
`senderOf` returns a userId, we'd prefer a user name. `lightUser` is available https://github.com/ornicar/lila/blob/master/modules/common/src/main/LightUser.scala#L6-L14